docs/providers/aws/guide/resources.md

```diff
@@ -12,15 +12,15 @@ layout: Doc
 
 # AWS Infrastructure Resources
 
-If you are using AWS as a provider for your Service, all [_Resources_](./intro.md#resources) are other AWS infrastructure resources which the AWS Lambda functions in your [_Service_](./intro.md#services) depend on, like AWS DynamoDB or AWS S3.
+If you are using AWS as a provider for your Service, all [_Resources_](./intro.md#resources) or other AWS infrastructure resources which the AWS Lambda functions depend are defined in your [_Service_](./intro.md#services) serverless definition file. For example, resources like AWS DynamoDB or AWS S3 can be created or updated.
```

Comment:
In think it should be further improved by changing: `or other AWS infrastructure resources which the` into `or other AWS infrastructure resources on which the`

Reply:
tweaked.